themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1580,29 +1580,24 @@ html body.tc-body.tc-single-tiddler-window {
 	overflow: auto;
 }
 
-.tc-tiddler-edit-frame .tc-dropzone-editor > div > div {
+.tc-grid {
 	display: grid;
 }
 
-.tc-tiddler-edit-frame .tc-dropzone-editor > div > div.tc-tiddler-preview {
-	grid-template-areas: 
-	"toolbar toolbar"
-	"editor preview";
+.tc-tiddler-frame .tc-grid.tc-tiddler-preview {
+	grid-template-areas: "toolbar toolbar" "editor preview";
```

Comment:
I do like `grid-template-area` --  I think for readability reasons we should use: 

```
.tc-tiddler-frame .tc-grid.tc-tiddler-preview {
	grid-template-areas: 
		"toolbar toolbar"
		"editor preview";
..
```

@Jermolene -- I know we normally do not use formatting like this, but I think it makes the "intent" with `grid-template-areas` so much easier to understand and maintain. -- What do you think?

Reply:
i made that more readable like you proposed